docs/spark/spark-procedures.md

```diff
@@ -417,10 +417,11 @@ To leave the original table intact while testing, use [`snapshot`](#snapshot) to
 
 #### Usage
 
-| Argument Name | Required? | Type | Description |
-|---------------|-----------|------|-------------|
-| `table`       | ✔️  | string | Name of the table to migrate |
-| `properties`  | ️   | map<string, string> | Properties for the new Iceberg table |
+| Argument Name        | Required? | Type                | Description                                    |
+|----------------------|-----------|---------------------|------------------------------------------------|
+| `table`              | ✔️        | string              | Name of the table to migrate                   |
+| `properties`         | ️         | map<string, string> | Properties for the new Iceberg table           |
+| `skip_corrupt_files` | ️         | boolean             | If true, skip corrupt files (false by default) |
```

Comment:
I think we need a better term than "corrupt" here. Maybe "skip on error"? Then the description is "If true, skips files which cannot be imported into Iceberg"?

Reply:
Changed in the latest commite.